packages/facebrok/PKGBUILD

```
@@ -3,13 +3,13 @@
 
 pkgname=facebrok
 pkgver=33.0f6fe8d
-pkgrel=1
+pkgrel=2
 pkgdesc='Social Engineering Tool Oriented to facebook.'
 groups=('blackarch' 'blackarch-social')
-arch=('any')
+arch=('x86_64' 'aarch64')
```

```suggestion
arch=('any')
```

this is not compiled